src/structures/GuildChannel.js

```diff
@@ -156,6 +156,15 @@ class GuildChannel extends Channel {
     return [...sorted.values()].indexOf(sorted.get(this.id));
   }
 
+  /**
+   * Checks if the channel is not viewable by users with only the @everyone role.
+   * @type {boolean} true if private, false otherwise.
+   */
+  get private() {
+    const { everyone } = this.guild.roles;
+    return !(this.permissionsFor(everyone)?.has(Permissions.FLAGS.VIEW_CHANNEL) ?? false);
```

Comment:
Unless roles are being swept, which we do not support, `permissionsFor` will never return null.
```suggestion
    return this.permissionsFor(everyone).has(Permissions.FLAGS.VIEW_CHANNEL);
```

Reply:
@kyranet probably wanted to suggest this instead:
```suggestion
    return !this.permissionsFor(everyone).has(Permissions.FLAGS.VIEW_CHANNEL);
```

Otherwise, the method would return `true` if the channel was public

Reply:
Fixed, thanks @almeidx